### PR TITLE
update System.Data.SQLite and fix csproj.

### PR DESCRIPTION
**Pull request by rwmcintosh**
Updating a dependency on System.Data.SQlite.core

### CODE REVIEW

File: src/PKSim.BatchTool/PKSim.BatchTool.csproj [hunk 1]
```diff
@@ -58,14 +55,21 @@
     </None>
   </ItemGroup>
 
+  <PropertyGroup>
+    <ContentSQLiteInteropFiles>true</ContentSQLiteInteropFiles>
+    <CopySQLiteInteropFiles>false</CopySQLiteInteropFiles>
+    <CleanSQLiteInteropFiles>false</CleanSQLiteInteropFiles>
+    <CollectSQLiteInteropFiles>false</CollectSQLiteInteropFiles>
+  </PropertyGroup>
+
```
Comment (rwmcintosh) [inline src/PKSim.BatchTool/PKSim.BatchTool.csproj:64]: These are properties added by System.Data.SQLite.Core when you add as a dependency.

When you set the Content task true, this is the task:

```
  <ItemGroup Condition="'$(ContentSQLiteInteropFiles)' != '' And
                        '$(ContentSQLiteInteropFiles)' != 'false' And
                        '@(SQLiteInteropFiles)' != ''">
    <Content Include="@(SQLiteInteropFiles)">
      <Link>%(RecursiveDir)%(FileName)%(Extension)</Link>
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </Content>
  </ItemGroup>
```

Pretty much what we had done manually before.

File: tests/PKSim.Matlab.Tests/PKSim.Matlab.Tests.csproj [hunk 2]
```diff
@@ -74,5 +71,16 @@
...
+    <CopySQLiteInteropFiles>false</CopySQLiteInteropFiles>
+    <CleanSQLiteInteropFiles>false</CleanSQLiteInteropFiles>
+    <CollectSQLiteInteropFiles>false</CollectSQLiteInteropFiles>
+  </PropertyGroup>
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
+    <ItemGroup>
+      <WindowsFiles Include="$(TargetDir)x64/SQLite.Interop.dll" />
+    </ItemGroup>
+    <Copy SourceFiles="@(WindowsFiles);" DestinationFolder="$(TargetFolder)" DestinationFiles="@(WindowsFiles->Replace('x64/SQLite.Interop.dll', 'SQLite.Interop.dll'))" />
+  </Target>
```
Comment (rwmcintosh) [inline tests/PKSim.Matlab.Tests/PKSim.Matlab.Tests.csproj:85]: For tests, we need to get this SQLite.Interop.dll into the root. This just copies the x64 version up one level to the output folder

Comment (rwmcintosh) [reply tests/PKSim.Matlab.Tests/PKSim.Matlab.Tests.csproj:85]: For now, no test projects are running on Linux, and other than debugging, we don't really need to build on Linux anyway.

**Comment by rwmcintosh** [comment]
This is going to fix the problem we had with SQLite on ubuntu when using RStudio

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**